### ISSUE TITLE
Is person detector used before training starts?

### ISSUE THREAD

**Issue by ChongMingWei** [leoxiaobin/deep-high-resolution-net.pytorch#236]
Hello, I trace the [training code](https://github.com/leoxiaobin/deep-high-resolution-net.pytorch/blob/master/tools/train.py) and the find that the dataloader loads the data from folder `'data/coco/'`.
I'm wondering that if the data in this folder is preprocessed from the original COCO dataset (by using the person detector) or not? 
Or there is something I missed, like the detection code in somewhere.
Thanks for your reply beforehand!

**Comment by wanghaichuan941221** [comment]
Yes, this is a top down method. Please check https://towardsdatascience.com/overview-of-human-pose-estimation-neural-networks-hrnet-higherhrnet-architectures-and-faq-1954b2f8b249 for more information
